src/test/scala/nl.knaw.dans.easy.deposit/docs/DDMSpec.scala

```diff
@@ -537,6 +537,10 @@ class DDMSpec extends TestSupportFixture with DdmBehavior {
           </boundedBy>
         </dcx-gml:spatial>
         <dcterms:license xsi:type="dcterms:URI">http://creativecommons.org/publicdomain/zero/1.0</dcterms:license>
+        <dcterms:language xsi:type="dcterms:ISO639-2">English</dcterms:language>
```

Comment:
I expect `eng` here, just as @lindareijnhoudt pointed out on `nld` below, but it should be using the object's key, rather than the value.